rosbag2_transport/include/rosbag2_transport/record_options.hpp

```diff
@@ -30,13 +30,16 @@ namespace rosbag2_transport
 struct RecordOptions
 {
 public:
-  bool all = false;
+  bool all_topics = false;
+  bool all_services = false;
   bool is_discovery_disabled = false;
   std::vector<std::string> topics;
+  std::vector<std::string> services;  // service event topic
   std::string rmw_serialization_format;
   std::chrono::milliseconds topic_polling_interval{100};
   std::string regex = "";
-  std::string exclude = "";
+  std::string exclude_topics = "";
+  std::string exclude_services = "";
```

Comment:
Comments from MichaelOrlov

I have a proposal about new parameters for topic/services exclusions:

Keep the old parameter exclude but rename it to the exclude_regex. it will be applicable to both topics and services.
Make exclude_topics and exclude_services as std::vector<std::string> i.e. space separated list of topics and services names for CLI arguments.
IMO it will be more user-friendly. We have had recently an incoming issue from one of the users when it was expected that the exclude parameter is a list of topics. Like opposite to the topics parameter.
It looks intuitively reasonable to expect this way.

Reply:
Okay. I will update the design https://github.com/ros2/rosbag2/blob/rolling/docs/design/rosbag2_record_replay_service.md and then modify code.